runtime/src/iree/builtins/ukernel/arch/arm_64/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ iree_bitcode_library(
     ukernel_bitcode_arm_64_entry_points
   ARCH
     wasm_64
+  INTERNAL_HDRS
+    "common_arm_64.h"
```

Comment:
> This commit goes toward addressing this problem by adding direct header dependencies.

Can you clarify what "this problem" is specifically? (I want to make sure we're solving the right thing, and deeper build restructuring may let us do that more easily)

Reply:
I was also wondering.  Is this about ensuring that dependent targets are properly rebuilt when a header is changed?

Reply:
That's it yes. I hade updated mmt4d.h locally and it didn't rebuild the microkernel. I was getting wrong output and it took me a bit too long for me to figure it out. BTW I think I managed to solve the limitation of this patch and will be able to handle filegroup. Stay tuned, hopefully I'll upload a revised version on Monday.

Reply:
Update: I have a patch that works but I need to make it more robust. Shouldn't be long.